src/20250120_Multi_FebruaryFundingUpdate/AaveV3Ethereum_FebruaryFundingUpdate_20250120.sol

```diff
@@ -0,0 +1,363 @@
+        fromUnderlyingPriceFeed: DPI_FEED,
+        toUnderlyingPriceFeed: AaveV3EthereumAssets.WETH_ORACLE,
+        amount: type(uint256).max,
+        slippage: 400
+      })
+    );
+
+    AaveV3Ethereum.COLLECTOR.swap(
+      MiscEthereum.AAVE_SWAPPER,
+      CollectorUtils.SwapInput({
+        milkman: MILKMAN,
+        priceChecker: PRICE_CHECKER,
+        fromUnderlying: AaveV3EthereumAssets.LUSD_UNDERLYING,
+        toUnderlying: AaveV3EthereumAssets.WETH_UNDERLYING,
+        fromUnderlyingPriceFeed: LUSD_FEED,
+        toUnderlyingPriceFeed: AaveV3EthereumAssets.WETH_ORACLE,
+        amount: type(uint256).max,
+        slippage: 400
+      })
+    );
+  }
+
+  function _withdrawAndSwapToUSDS() internal {
+    AaveV3Ethereum.COLLECTOR.withdrawFromV2(
+      CollectorUtils.IOInput({
+        pool: address(AaveV2Ethereum.POOL),
+        underlying: AaveV2EthereumAssets.DAI_UNDERLYING,
+        amount: IERC20(AaveV2EthereumAssets.DAI_A_TOKEN).balanceOf(
+          address(AaveV3Ethereum.COLLECTOR)
+        ) - 1 ether
+      }),
+      address(AaveV3Ethereum.COLLECTOR)
+    );
+
+    AaveV3Ethereum.COLLECTOR.withdrawFromV3(
+      CollectorUtils.IOInput({
+        pool: address(AaveV3Ethereum.POOL),
+        underlying: AaveV3EthereumAssets.DAI_UNDERLYING,
+        amount: IERC20(AaveV3EthereumAssets.DAI_A_TOKEN).balanceOf(
+          address(AaveV3Ethereum.COLLECTOR)
+        ) - 1 ether
+      }),
+      address(AaveV3Ethereum.COLLECTOR)
+    );
+
+    AaveV3Ethereum.COLLECTOR.swap(
+      MiscEthereum.AAVE_SWAPPER,
+      CollectorUtils.SwapInput({
+        milkman: MILKMAN,
+        priceChecker: PRICE_CHECKER,
+        fromUnderlying: AaveV3EthereumAssets.DAI_UNDERLYING,
+        toUnderlying: AaveV3EthereumAssets.USDS_UNDERLYING,
+        fromUnderlyingPriceFeed: AaveV3EthereumAssets.DAI_ORACLE,
+        toUnderlyingPriceFeed: AaveV3EthereumAssets.USDS_ORACLE,
+        amount: type(uint256).max,
+        slippage: 75
+      })
+    );
+  }
+
+  function _swap() internal {
+    AaveV3Ethereum.COLLECTOR.swap(
+      MiscEthereum.AAVE_SWAPPER,
+      CollectorUtils.SwapInput({
+        milkman: MILKMAN,
+        priceChecker: PRICE_CHECKER,
+        fromUnderlying: AaveV3EthereumAssets.USDC_UNDERLYING,
+        toUnderlying: AaveV3EthereumAssets.GHO_UNDERLYING,
+        fromUnderlyingPriceFeed: AaveV3EthereumAssets.USDC_ORACLE,
+        toUnderlyingPriceFeed: GHO_USD_FEED,
+        amount: AMOUNT_SWAP_USDC,
+        slippage: 75
+      })
+    );
+
+    AaveV3Ethereum.COLLECTOR.swap(
+      MiscEthereum.AAVE_SWAPPER,
+      CollectorUtils.SwapInput({
+        milkman: MILKMAN,
+        priceChecker: PRICE_CHECKER,
+        fromUnderlying: AaveV3EthereumAssets.USDT_UNDERLYING,
+        toUnderlying: AaveV3EthereumAssets.GHO_UNDERLYING,
+        fromUnderlyingPriceFeed: AaveV3EthereumAssets.USDT_ORACLE,
+        toUnderlyingPriceFeed: GHO_USD_FEED,
+        amount: AMOUNT_SWAP_USDT,
+        slippage: 75
+      })
+    );
+  }
+
```

Comment:
Isn't the logic missing to deposit gho to prime instance?

Reply:
Since we are swapping assets via cowswap, we cannot deposit the swap output in this payload